<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>320</width><height>180</height></rect>
  </property>
  <property name="windowTitle"><string>Login</string></property>
  <widget class="QLineEdit" name="lineEdit_usuario">
   <property name="geometry"><rect><x>90</x><y>30</y><width>150</width><height>25</height></rect></property>
   <property name="placeholderText"><string>Usu√°rio</string></property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_senha">
   <property name="geometry"><rect><x>90</x><y>70</y><width>150</width><height>25</height></rect></property>
   <property name="echoMode"><enum>QLineEdit::Password</enum></property>
   <property name="placeholderText"><string>Senha</string></property>
  </widget>
  <widget class="QPushButton" name="pushButton_login">
   <property name="geometry"><rect><x>110</x><y>120</y><width>100</width><height>30</height></rect></property>
   <property name="text"><string>Entrar</string></property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
